standard/external_media_links/external_media_list.lua

```diff
@@ -0,0 +1,275 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:ExternalMediaList
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Flag = require('Module:Flags')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Tabs = require('Module:Tabs')
+
+local MediaList = {}
+
+local MAX_NUMBER_OF_STORED_PER_KEY = 10
```

Comment:
the EML template stored up to 16 already, are we fine with 10?

Reply:
we can adjust it